src/Lime.Messaging/Contents/Reply.cs

```diff
@@ -72,5 +73,26 @@ public class InReplyTo
         /// </value>
         [DataMember(Name = VALUE_KEY)]
         public Document Value { get; set; }
+
+        /// <summary>
+        /// Indicates the direction of the message in the thread.
+        /// </summary>
+        [DataMember(Name = DIRECTION_KEY)]
+        public MessageDirection Direction { get; set; }
```

Comment:
Is this the direction of the "Reply"  or is it the direction of the RepliedMessage?